packages/react-select/src/Select.js

```diff
@@ -73,8 +75,41 @@ type FormatOptionLabelMeta = {
   inputValue: string,
   selectValue: ValueType,
 };
+export type AccessibilityProp = {
```

Comment:
What do you think about moving these to a separate file?

Reply:
I think you are right! I'll try to find a good place for them.